openedx/core/djangoapps/user_api/views.py

```diff
@@ -161,3 +163,13 @@ class CountryTimeZoneListView(generics.ListAPIView):
     def get_queryset(self):
         country_code = self.request.GET.get("country_code", None)
         return get_country_time_zones(country_code)
+
+
+class DisableUserListView(generics.ListAPIView):
```

Comment:
same comment as above, using the adjective rather than verb form, ie `Disabled`.

Reply:
updated